### PR TITLE
Crawling people who aren't full-stunned (able to use items or move) now get hit by projectiles

### PR DESCRIPTION
**Pull request by silicons**
Oof, how did I think this was a good idea...
@optimumtact

### CODE REVIEW

**Comment by evsey9** [comment]
bad change, if you're 400% slower you should have SOME advantage

**Comment by evsey9** [comment]
also makes zero sense because why would you suddenly become unhittable when stunned but crawling makes projectiles hit you

**Comment by optimumtact** [comment]
evsey9 on the ignore list for bad ideas

**Comment by Iamgoofball** [comment]
does this also fix wavedashing? nice

**Comment by evsey9** [comment]
I didn't know nanotrasen made all their guns have HOMING BULLETS

**Comment by Iamgoofball** [comment]
@evsey9 its the year 2525

**Comment by ExcessiveUseOfCobblestone** [comment]
> going prone doesn't up your chances of not being hit

This is such a bad top-down i'm going back to diablo

**Comment by Kierany9** [comment]
>removing matrix-ing

why?

**Comment by subject217** [comment]
the wrong fix for crawling.

it should just be a lot slower to deter people from using it tactically, and have a longer delay to get up.

File: code/modules/projectiles/projectile.dm [hunk 1]
```diff
@@ -568,9 +568,15 @@
...
 /obj/item/projectile/Crossed(atom/movable/AM) //A mob moving on a tile with a projectile is hit by it.
-	..()
-	if(isliving(AM) && (AM.density || AM == original) && !(src.pass_flags & PASSMOB))
-		Bump(AM)
+	. = ..()
+	if(isliving(AM) && !(pass_flags & PASSMOB))
+		var/mob/living/L = AM
+		if(AM == original)		//specific aiming for
+			Bump(AM)
+		if(AM.density)			//blocks projectiles
```
Comment (deathride58) [inline code/modules/projectiles/projectile.dm]: Uh, kev, this makes bullets pass through walls and other solid objects. Move this and the previous `if()` check right before it to somewhere before the `if(isliving())` check

**Comment by PKPenguin321** [comment]
@Iamgoofball 
>does this also fix wavedashing? nice

was already pretty much fixed by the rest delay but yeah it seems to make it even worse

**Comment by subject217** [comment]
>was already pretty much fixed by the rest delay but yeah it seems to make it even worse

if by "even worse" you mean "impossible" yeah it does do that

**Comment by 81Denton** [comment]
Is there a sort of way to distinguish horizontal people you can hit from horizontal people you can't hit?

**Comment by subject217** [comment]
visually no, this would mostly matter at beyond visual range though since people who are stunned *shouldn't* be moving around

**Comment by zamolxius** [comment]
just make them 50% chance to be hit when on floor

**Comment by ExcessiveUseOfCobblestone** [comment]
Where are the realism people talking about how prone makes you harder to hit.

**Comment by evsey9** [comment]
Then make crawling slower instead of making it stupid and illogical?

**Comment by ExcessiveUseOfCobblestone** [comment]
that is subject's original suggestion, yes.

**Comment by optimumtact** [comment]
I appreciate your suggestions but they are wrong

**Comment by optimumtact** [comment]
trust me when the slip rework finishes this will be considered a good change

**Comment by evsey9** [comment]
if you think opinions can be wrong then you are an idiot

**Comment by optimumtact** [comment]
they're not my opinions therefore they're wrong, sorry kid

**Comment by nicbn** [comment]
Just aim for the body, they are alredy slow as fug

File: code/modules/projectiles/projectile.dm (hunk 1 above)
Comment (kevinz000) [reply code/modules/projectiles/projectile.dm]: What? Previous code already checked for living before trying to force bump, you sure?

**Comment by optimumtact** [comment]
how hard is it?

**Comment by Therapist1** [comment]
not very hard unless you’re long range, then it’s pretty difficult

**Comment by subject217** [comment]
I'd say it's a fair PITA if they know what they're doing. It's doubled because they can ALSO disarm spam you while they are doing this.

**Comment by anconfuzedrock** [comment]
I dislike this because I feel it makes crawling incredibly similar to walking. Even before crawling I’ve enjoyed resting to play dead, but now crawling is just “walking but you can go through flaps.”

**Comment by Robustin** [comment]
I know TGstation loves its bad combat memes but TACTICAL CRAWLING is just bad.

**Comment by subject217** [comment]
The point of crawling isn't to crawl around all the time like a jackass, it's to give a form of CC that doesn't involve totally 100% immobilizing you. Still think this is a bad change but saying "THIS MAKES CRAWLING USELESS" is stupid as hell. Did you even read the actual stun rework thread on the forums?

**Comment by optimumtact** [comment]
of course they didn't

**Pull request merged**